diff -urNp -x '*.orig' guile-2.2.7.org/libguile/guile-func-name-check guile-2.2.7/libguile/guile-func-name-check
--- guile-2.2.7.org/libguile/guile-func-name-check	2017-04-20 19:02:45.000000000 +0200
+++ guile-2.2.7/libguile/guile-func-name-check	2022-02-26 00:01:59.308490795 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/awk -f
+#!/bin/awk -f
 #
 #  Copyright (C) 2000, 2001, 2006, 2017 Free Software Foundation, Inc.
 # 
